src/hb-ot-shape.cc

```diff
@@ -632,7 +631,7 @@ hb_ot_substitute_complex (hb_ot_shape_context_t *c)
 
   c->plan->substitute (c->font, buffer);
 
-  if (0) /* XXX Call morx instead. */
+  if ((0)) /* XXX Call morx instead. */
```

Comment:
Apparently the way clang prefers for disabled codes.

Reply:
The same here https://stackoverflow.com/a/29177815 and apparently like that instead